plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/BaseMySqlConnectorTest.java

```diff
@@ -38,12 +41,121 @@
 import static org.testng.Assert.assertFalse;
 import static org.testng.Assert.assertTrue;
 
-abstract class BaseMySqlIntegrationSmokeTest
-        // TODO extend BaseConnectorTest
-        extends AbstractTestIntegrationSmokeTest
+abstract class BaseMySqlConnectorTest
+        extends BaseConnectorTest
 {
     protected TestingMySqlServer mysqlServer;
```

Comment:
I think we should have a abstract getter for this field. This resource is not controlled here so I don't think it should be stored here. WDYT?

Reply:
Preexistent - but I can change that